README.md

```diff
@@ -27,6 +27,10 @@ Using Pipenv
 > pipenv install --three
 ```
 
+#### Windows 10 Setup
+
+To install the project's Python dependencies on Windows 10, you must run `scripts\install_windows.bat`
+
```

Comment:
I think we should go with the official way of [installing Twisted on Windows](https://twistedmatrix.com/trac/wiki/Downloads#ThroughPip)

```shell
> pip install Twisted[windows_platform]
```

This whole block should be moved before the `Using Pip` line as a `Windows pre- requisites` section. Twisted should be installed before the regular requirements.